src/Whoops/Resources/css/whoops.base.css

```diff
@@ -379,9 +397,9 @@ pre.prettyprint, code.prettyprint {
 }
 
 #help-exc-message {
-	position: absolute;
-	left: 65%;
-	top: 10px;
+    position: absolute;
+    left: 52%;
```

Comment:
That's moving the exception *help* to the left, to make room for the hider help message. Originally, there was no room (on my screen, anyway, ~1260px max width) for the hider message in the upper corner. On a smaller screen (ex. 1024 MW), they juuuust approach overlapping, because the exception help is using a percentage for its position.